rs/nns/sns-wasm/src/sns_wasm.rs

```diff
@@ -27,6 +27,10 @@ use ic_cdk::api::stable::StableMemory;
 use ic_nervous_system_clients::canister_id_record::CanisterIdRecord;
 use ic_nervous_system_common::{ONE_TRILLION, SNS_CREATION_FEE};
 use ic_nervous_system_proto::pb::v1::Canister;
+use ic_nns_constants::{
+    DEFAULT_SNS_GOVERNANCE_CANISTER_WASM_MEMORY_LIMIT,
+    DEFAULT_SNS_NON_GOVERNANCE_CANISTER_WASM_MEMORY_LIMIT,
```

Comment:
I don't see where this is defined

Reply:
Sorry, I forgot to commit that file, fixed